packages/ckeditor5-emoji/src/emojimention.ts

```diff
@@ -222,8 +222,7 @@ export default class EmojiMention extends Plugin {
 
 			const emojis: Array<MentionFeedObjectItem> = this._emojiDatabasePlugin.getEmojiBySearchQuery( searchQuery )
 				.map( emoji => {
-					// TODO: The configuration `emoji.skinTone` option is ignored here.
-					let text = emoji.skins.default;
+					let text = emoji.skins[ this.editor.config.get( 'emoji.skinTone' ) || 'default' ];
```

Comment:
This way, the plugin tries to read a property that might not exist in the configuration (when `EmojiPicker` is not loaded).

Please move the `config.define()` call from `EmojiPicker` to `EmojiDatabase`, and then:

```suggestion
					let text = emoji.skins[ this.editor.config.get( 'emoji.skinTone' ) ] || emoji.skins.default;
```

Reply:
We store a configuration property as a variable in L75. To be consistent in the plugin, we could do the same for the skin tone property.

https://github.com/ckeditor/ckeditor5/blob/4033ee170e918be9668ae42a17d803f6c88f0b2c/packages/ckeditor5-emoji/src/emojimention.ts#L75

Reply:
Fixed in [9f6423c](https://github.com/ckeditor/ckeditor5/pull/17800/commits/9f6423c37bd9ac79543f5663e8b361fa9b933d44)